3-reveal/migrations/2-transaction_tables/deploy/plan_opensrpeventid_field.psql

```diff
@@ -0,0 +1,12 @@
+-- Deploy reveal_transaction_tables:plan_opensrpeventid_field to pg
+-- requires: plans
+
+BEGIN;
+
+SET search_path TO :"schema",public;
+
+-- adds fi_opensrpeventid field
+ALTER TABLE plans
+ADD COLUMN fi_opensrpeventid VARCHAR NULL;
```

Comment:
Can we add an index on this field `fi_opensrpeventid`?  We are filtering by it and so it makes sense to add an index to make the query perform better.